data/hai/unfettered 1 alphas.txt

```diff
@@ -977,11 +985,16 @@ mission "<uhai> 7: Cavalry Arrives"
 
 mission "<uhai> 8: Unfettered Humans?"
 	landing
-	name "Who are the 'unfettered humans'?"
+	minor
+	name "Learn more about the 'unfettered humans'?"
 	description `The Auxiliary was manned by 'unfettered humans'. Find out what this means, pirates may have the answers.`
 	destination "Darkcloak"
 	to offer
-		has "<uhai> 5: Scan Auxiliary: done"
+		has "<uhai> 5: Auxiliary Scans: done"
+	on offer
+		conversation
+			`You learned from the Unfettered Hai that the Auxiliary you fought recently in order to get the scans was in fact manned by 'unfettered humans'. It would be wise to find out what that means, and where they come from, especially considering they shot you on sight.`
```

Comment:
```suggestion
			`You learned from the Unfettered Hai that the Auxiliary you fought recently in order to get the scans was in fact manned by 'unfettered humans'. It would be wise to find out what that means, and where they come from, especially considering they shot at you on sight.`
```